libcst/metadata/base_provider.py

```diff
@@ -75,7 +82,11 @@ def _gen_impl(self, module: "Module") -> None:
         """
         ...
 
-    def set_metadata(self, node: "CSTNode", value: _T) -> None:
+    # pyre-ignore[46]: The covariant `value` isn't type-safe because we write it to
+    # pyre: `self._computed`, however we assume that only one subclass in the MRO chain
+    # pyre: will every call `set_metadata`, so it's okay for our purposes. There's no
```

Comment:
The `every` seems redundant?
```suggestion
    # pyre: will call `set_metadata`, so it's okay for our purposes. There's no
```